src/alembic_utils/replaceable_entity.py

```diff
@@ -25,14 +25,17 @@ def normalize_whitespace(text, base_whitespace: str = " ") -> str:
 
 
 @contextmanager
-def simulate_entity(connection, entity):
+def simulate_entity(connection, entity, setup=None):
```

Comment:
This setup param appears to only be useful during testing. I'd prefer to assume that the correct setup already exists when the migration is being autogenerated and have the migration fail if that assumption is incorrect